### ISSUE TITLE
New delegation when no cTypes

### ISSUE THREAD

**Issue by aenyway** [KILTprotocol/demo-client#122]
the current behavior of "new delegation" on cType oder delegation list view is unsatisfying. right now just nothing happens, there should be a message after pressing the button/action

**Comment by flechtnergalani** [comment]
I agree we should at least be some text in the ctype selector saying that no CTYPEs were found. The current popup is just empty with an active OK button (this should be inactive).  
![image](https://user-images.githubusercontent.com/39338561/79569931-da1ea280-80b8-11ea-8a66-77e28d9ff156.png)

What is worse, clicking the button actually seems to trigger invalid behaviour:
![image](https://user-images.githubusercontent.com/39338561/79570132-2cf85a00-80b9-11ea-99a2-0f7a9c0a9489.png)
